public/js/export.js

```diff
@@ -39,49 +39,57 @@ $(function() {
     });
 
     $("#data-creds").dialog({
-            autoOpen : false,
-            modal : true,
-            open : function() {
+        autoOpen : false,
+        modal : true,
+        open : function() {
+            checkInputs();
+            $("#data-creds").keyup(function(e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this function so that its implementation is not identical to the one on line 20. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=javascript%3AS4144)